src/main/java/org/jabref/logic/importer/fileformat/BibtexParser.java

```diff
@@ -223,10 +218,11 @@ private ParserResult parseFileContent() throws IOException {
                 // Not a comment, preamble, or string. Thus, it is an entry
                 parseAndAddEntry(entryType);
             }
-
```

Comment:
Why was this original blank line deleted?

Reply:
Probably because we added a logging statement here then removed the line completely

Reply:
I still think this should be re-added

Reply:
As I said during the meeting, I did re-add it but for some reason it did not show up here on GitHub

Reply:
Nevermind, I just checked and it's not there, I could have sworn I fixed it though. I can make another PR and add the whitespace!